test/specs/components/Settings.spec.jsx

```diff
@@ -94,17 +85,4 @@ describe('Settings', () => {
         const description = TestUtils.scryRenderedDOMComponentsWithClass(component, 'settings-description')[0];
         description.textContent.should.eq(mockedStore.getState().ui.text.notificationSettingsChannelsDescription);
     });
-
-    describe('No user id', () => {
```

Comment:
Hmm, surprised this simply gets removed Should this test expectation simply be changed to `should render` ?